src/core/lib/slice/slice_intern.cc

```diff
@@ -39,14 +41,7 @@
 #define TABLE_IDX(hash, capacity) (((hash) >> LOG2_SHARD_COUNT) % (capacity))
 #define SHARD_IDX(hash) ((hash) & ((1 << LOG2_SHARD_COUNT) - 1))
 
-typedef struct interned_slice_refcount {
-  grpc_slice_refcount base;
-  grpc_slice_refcount sub;
-  size_t length;
-  gpr_atm refcnt;
-  uint32_t hash;
-  struct interned_slice_refcount* bucket_next;
-} interned_slice_refcount;
+typedef grpc_core::InternedSliceRefcount interned_slice_refcount;
```

Comment:
I can understand that this is to minimize the diff, but it looks odd to typedef something with C++-style naming/behavior to something with C-style naming/behavior.

Reply:
I think for now it's an easier diff to review if we leave the names as is when able (so we don't have a million lines to change added)